include/cantera/thermo/MixtureFugacityTP.h

```diff
@@ -492,14 +492,6 @@ class MixtureFugacityTP : public ThermoPhase
     virtual void getActivityConcentrations(double* c) const;
 
 protected:
```

Comment:
Is the thinking here that the removed virtual method is simply unnecessary, given that `PengRobinson::pressure()` exists?

Reply:
Right, the `pressureCalc` function wasn't implemented by either of the derived classes, and was also never called from anywhere.